### PR TITLE
New package: DassaultSystemes.Draftsight version 24.0.0.5001

### PR DESCRIPTION
**Pull request by khaffner**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137419)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 694.)

**Comment by khaffner** [comment]
Hm, I assumed the parameter here would work
https://silentinstallhq.com/draftsight-2023-silent-install-how-to-guide/

These are the parameters as told by the installer itself:
image

Running it with just /Q starts off good, then this pops up not so quietly:
image
I guess this is why we need to inject LICENSETYPE=0 somehow. 

Does anyone have an idea?

**Comment by stephengillie** [comment]
Hi @khaffner,

This appears to be an ephemeral pipeline issue, not a package issue. Sorry for any concern caused.

**Comment by stephengillie** [comment]
Manual Validation encountered:
image

File: manifests/d/DassaultSystemes/Draftsight/24.0.0.5001/DassaultSystemes.Draftsight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=NVS0.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: DassaultSystemes.Draftsight
+PackageVersion: 24.0.0.5001
+InstallerType: exe
+InstallerSwitches:
+  Silent: /Q /C:"setup.exe /S /v""LICENSETYPE=0 /qn"""
+  SilentWithProgress: /Q /C:"setup.exe /S /v""LICENSETYPE=0 /qb"""
+UpgradeBehavior: install
```
Comment (stephengillie) [inline manifests/d/DassaultSystemes/Draftsight/24.0.0.5001/DassaultSystemes.Draftsight.installer.yaml:10]:
```suggestion
UpgradeBehavior: install
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Comment by stephengillie** [comment]
> Hm, I assumed the parameter here would work https://silentinstallhq.com/draftsight-2023-silent-install-how-to-guide/
> 
> These are the parameters as told by the installer itself: image
> 
> Running it with just /Q starts off good, then this pops up not so quietly: image I guess this is why we need to inject LICENSETYPE=0 somehow.
> 
> Does anyone have an idea?

With the dependency installed, I reached the same situation.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  2024-02-02 19:19:28.032 [FAIL] WindowsPackageManager.dll!00007FFA6889BDD2: ReturnHr(1) tid(47c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FFA6888203A: (caller: 00007FFA68767669) Exception(1) tid(47c) 80190193 Forbidden (403). 2024-02-02 19:19:28.032 [FAIL] WindowsPackageManager.dll!00007FFA68723AF1: LogHr(1) tid(47c) 80190193 Forbidden (403). 2024-02-02 19:19:28.032 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(53)\WindowsPackageManager.dll!00007FFA688A7301: (caller: 00007FFA687704FE) LogHr(2) tid(47c) 80190193 Forbidden (403).     Msg:[winrt::hresult_error: Forbidden (403).]  2024-02-02 19:19:28.032 [REPO] Source add/update failed, waiting a bit and retrying: winget 2024-02-02 19:19:30.058 [FAIL] WindowsPackageManager.dll!00007FFA6889BDD2: ReturnHr(2) tid(47c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FFA6888203A: (caller: 00007FFA68767669) Exception(2) tid(47c) 80190193 Forbidden (403). 2024-02-02 19:19:30.058 [FAIL] WindowsPackageManager.dll!00007FFA68723AF1: LogHr(3) tid(47c) 80190193 Forbidden (403). 2024-02-02 19:19:30.058 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(540)\WindowsPackageManager.dll!00007FFA688A68E3: (caller: 00007FFA68668B8D) LogHr(4) tid(47c) 80190193 Forbidden (403). 2024-02-02 19:19:30.058 [REPO] Failed to update source: winget 2024-02-02 19:19:30.235 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FFA688A6B53: (caller: 00007FFA68668B8D) LogHr(5) tid(47c) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFA687B9298: (caller: 00007FFA688724F9) Exception(3) tid(47c) 8A15005E ]  2024-02-02 19:19:30.235 [REPO] Failed to open available source: msstore 2024-02-02 19:19:30.247 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FFA6878453A: (caller: 00007FFA68770B9E) Exception(4) tid(47c) 8A15000F  2024-02-02 19:19:30.247 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FFA688A6B53: (caller: 00007FFA68668B8D) LogHr(6) tid(47c) 8A15000F     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FFA6878453A: (caller: 00007FFA68770B9E) Exception(4) tid(47c) 8A15000F ]  2024-02-02 19:19:30.247 [REPO] Failed to open available source: winget 2024-02-02 19:19:30.247 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(571)\WindowsPackageManager.dll!00007FFA687710DB: (caller: 00007FFA68668B8D) Exception(5) tid(47c) 8A15004B  2024-02-02 19:19:31.958 [FAIL] WindowsPackageManager.dll!00007FFA6809BDD2: ReturnHr(1) tid(adc) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FFA6808203A: (caller: 00007FFA67F67669) Exception(1) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:31.958 [FAIL] WindowsPackageManager.dll!00007FFA67F23AF1: LogHr(1) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:31.958 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(53)\WindowsPackageManager.dll!00007FFA680A7301: (caller: 00007FFA67F704FE) LogHr(2) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:31.958 [REPO] Source add/update failed, waiting a bit and retrying: winget 2024-02-02 19:19:33.983 [FAIL] WindowsPackageManager.dll!00007FFA6809BDD2: ReturnHr(2) tid(adc) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FFA6808203A: (caller: 00007FFA67F67669) Exception(2) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:33.983 [FAIL] WindowsPackageManager.dll!00007FFA67F23AF1: LogHr(3) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:33.983 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(540)\WindowsPackageManager.dll!00007FFA680A68E3: (caller: 00007FFA67E68B8D) LogHr(4) tid(adc) 80190193 Forbidden (403). 2024-02-02 19:19:33.983 [REPO] Failed to update source: winget 2024-02-02 19:19:34.047 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FFA680A6B53: (caller: 00007FFA67E68B8D) LogHr(5) tid(adc) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFA67FB9298: (caller: 00007FFA680724F9) Exception(3) tid(adc) 8A15005E ]  2024-02-02 19:19:34.047 [REPO] Failed to open available source: msstore 2024-02-02 19:19:34.059 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FFA67F8453A: (caller: 00007FFA67F70B9E) Exception(4) tid(adc) 8A15000F  2024-02-02 19:19:34.059 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FFA680A6B53: (caller: 00007FFA67E68B8D) LogHr(6) tid(adc) 8A15000F     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FFA67F8453A: (caller: 00007FFA67F70B9E) Exception(4) tid(adc) 8A15000F ]  2024-02-02 19:19:34.059 [REPO] Failed to open available source: winget 2024-02-02 19:19:34.059 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(571)\WindowsPackageManager.dll!00007FFA67F710DB: (caller: 00007FFA67E68B8D) Exception(5) tid(adc) 8A15004B 

(Automated response - build 695.)

**Comment by stephengillie** [comment]
> Automatic Validation ended with: image

**Pull request closed without merging**